### DISCUSSION TITLE
Icon request: dock

### DISCUSSION THREAD

**Discussion by jwmickey** [FortAwesome/Font-Awesome#21205] [Icon Requests]
**What can it be used for?**
A set of "dock position" icons to indicate position of some element.  The example image below is a screenshot of the icons used in Chrome developer toolbar dock options.  It may be useful to also add a dock icon for "top" position, which is not included in the screenshot.

**Example image**
![example icons](https://user-images.githubusercontent.com/400616/59117968-0efc8d80-891d-11e9-9a75-a5756952112b.png)

**Requested icons**
dock-left
dock-right
dock-top
dock-bottom

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
